### ISSUE TITLE
Sharing MoE results for integration tests

### ISSUE THREAD

**Issue by younesbelkada** [google-research/t5x#874]
Hi there! 👋 

We have been working to add Switch transformers in Hugging Face `transformers` in https://github.com/huggingface/transformers/pull/19323 ! 
We believe that we have almost finalized the conversion process, and would like to ask you few questions to make sure the conversion has been done correctly. 
We are currently matching the logits between the output that is obtained with `t5x.infer` and our forward pass, given the same input. Therefore we have tried to play a bit with the models and do some generations. The generations can be done in a T5-style using `transformers` as follows:
```
from transformers import SwitchTransformersForConditionalGeneration, AutoTokenizer

tokenizer = AutoTokenizer.from_pretrained("t5-small")
model = SwitchTransformersForConditionalGeneration.from_pretrained("switch-base-8")

input_text = "summarize: Hello world"
input_ids = tokenizer(input_text, return_tensors="pt").input_ids
output = model.generate(input_ids)

print(tokenizer.decoder(output[0]))
>>> The best way to do it is to use a smartphone. Hello there
```
The model seems to output gibberish for T5 tasks such as translation, but seems to produce intelligible output for summarization (even though it is not great).  Are the weights that has been open-sourced for the Switch transformers pre-trained weights (ie not fine-tuned)? Therefore not "ready-to-use" as Flan-T5?
We are observing the same behavior for `t5_v1_1` models that have been open-sourced and converted, if my understanding is correct, Flan-T5 is the only model that is "ready-to-use" after downloading the weights, other models needs to be fine-tuned - therefore the output that we are currently getting with our converted MoE seems to be consitent/correct. 

Let us know if we are missing something! Would love to discuss further details and introduce together the first MoE model in `transformers` 
Thanks!
